VATSpy.dat

```diff
@@ -14376,7 +14376,9 @@ YBBH|Birri Beach|-16.491667|139.283333||YISA|0
 YBBL|Billabalong|-27.416389|115.833333||YOLW|0
 YBBN|Brisbane|-27.384167|153.1175|BN|YINL|0
 YBBN|Brisbane|-27.384167|153.1175|BN-S|YINL|0
-YBBN|Brisbane|327.384167|153.1175|BN-F|YINL|0
+YBBN|Brisbane|-27.384167|153.1175|BN-F|YINL|0
+YBBN|Brisbane|-27.384167|153.1175|BN-N|YINL|0
+YBBN|Brisbane|-27.384167|153.1175|BN-W|YINL|0

```

Comment:
While it has not been done in the existing file, please change the additional YBBN entries to have the pseudo id to 1.

**Example:**
```
YBBN|Brisbane|-27.384167|153.1175|BN|YINL|0
YBBN|Brisbane|-27.384167|153.1175|BN-S|YINL|1
YBBN|Brisbane|-27.384167|153.1175|BN-F|YINL|1
YBBN|Brisbane|-27.384167|153.1175|BN-N|YINL|1
YBBN|Brisbane|-27.384167|153.1175|BN-W|YINL|1
```